packages/core/content-releases/admin/src/components/ReleaseActionOptions.tsx

```diff
@@ -40,30 +40,51 @@ const FieldWrapper = styled(Field)<FieldWrapperProps>`
     text-transform: capitalize;
   }
 
-  &:active,
   &[data-checked='true'] {
-    color: ${({ theme }) => theme.colors.primary700};
-    background-color: ${({ theme }) => theme.colors.primary100};
-    border-color: ${({ theme }) => theme.colors.primary700};
+    color: ${({ theme, actionType }) =>
+      actionType === 'publish' ? theme.colors.primary700 : theme.colors.danger600};
+    background-color: ${({ theme, actionType }) =>
+      actionType === 'publish' ? theme.colors.primary100 : theme.colors.danger100};
+    border-color: ${({ theme, actionType }) =>
+      actionType === 'publish' ? theme.colors.primary700 : theme.colors.danger600};
   }
 
   &[data-checked='false'] {
     border-left: ${({ actionType }) => actionType === 'unpublish' && 'none'};
     border-right: ${({ actionType }) => actionType === 'publish' && 'none'};
   }
+
+  &[data-checked='false']:hover {
```

Comment:
What do you think about this change:

```suggestion
  &[data-checked='false'][data-disabled='false']:hover {
```

in that way we only add the hover when is not disabled

Reply:
yeah, it is something also Lucas highlighted, thanks Fernando